lib/Dialect/FIRRTL/Import/FIRParser.cpp

```diff
@@ -583,6 +587,24 @@ ParseResult FIRParser::parseIntLit(int32_t &result, const Twine &message) {
   return success();
 }
 
+/// versionLit    ::= version
+/// deconstruct a version literal into parts
+ParseResult FIRParser::parseVersionLit(int32_t major, int32_t minor,
```

Comment:
1) Maybe `uint32_t`?
2) These arguments appear unused, which since the version isn't used yet isn't a big deal but unclear what they're for-- are they meant to be a minimum version to check against or a default if one isn't found?

Reply:
yea, meant to be by referenfce.